spinn_front_end_common/interface/interface_functions/insert_edges_to_extra_monitor_functionality.py

```diff
@@ -67,7 +65,7 @@ def __call__(self, machine_graph, placements, machine,
             "Inserting edges between vertices which require FR speed up "
             "functionality.")
 
-        if application_graph is None:
+        if application_graph == 0:
```

Comment:
Probably want to check the property here

Reply:
good spot. test got it too Fixed.